build_library/extract-initramfs-from-vmlinuz.sh

```diff
@@ -65,37 +49,22 @@ if [[ ! -s "${image}" ]]; then
 fi
 mkdir -p "${out}"
 
-tmp=$(mktemp --directory /tmp/eifv-XXXXXX)
-trap "rm -rf ${tmp}" EXIT
+tmp=$(mktemp --directory eifv-XXXXXX)
+trap 'rm -rf -- "${tmp}"' EXIT
+ROOTFS_IDX=0
 
-tmp_dec="${tmp}/decompress"
-mkdir "${tmp_dec}"
-fr_prefix="${tmp}/first-round"
+# arm64 kernels are not compressed, so try decompressing once.
+# Other kernels are compressed, so also try decompressing twice.
+for OFF1 in $(find_xz_headers "${image}")
+do
+    decompress_at "${image}" "${OFF1}" > "${tmp}/initrd.cpio.xz"
+    try_extract "${tmp}/initrd.cpio.xz"
```

Comment:
Maybe call this `${tmp}/initrd.cpio.maybe_xz`?

Reply:
Okay, I did consider that. It's not like extensions mean much in Linux anyway!

Reply:
Yeah, these work for me. Thanks.